cmd/pr.go

```diff
@@ -0,0 +1,69 @@
+package cmd
+
+import (
+	"bufio"
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/fatih/color"
+	"github.com/spf13/cobra"
+)
+
+var compareBranch string
+var title string
+var description string
+
+func init() {
+	rootCmd.AddCommand(prCmd)
+	prCmd.Flags().StringVar(&compareBranch, "compare", "", "The branch to compare with base branch. Defaults to current local branch.")
+	prCmd.Flags().StringVar(&title, "title", "", "The title of the PR.")
+	prCmd.Flags().StringVar(&description, "description", "", "The description of the PR.")
+	prCmd.MarkFlagRequired("title")
+}
+
+var prCmd = &cobra.Command{
+	Use:   "pr",
+	Short: "Create a pull request [github]",
+	Long:  `Create a pull request on github from compare branch to base branch`,
+	Run: func(cmd *cobra.Command, args []string) {
+		createPR()
```

Comment:
you should do some input validation i general. the Command has pre run functions with error PreRunE which can be used for this.